### PR TITLE
Issue #5466 Files directory htaccess does not match text in core/includes/file.inc 539-555

### LINKED ISSUE #5466: Files directory htaccess does not match text in core/includes/file.inc 539-555
## Description of the bug

The .htaccess in the files folder in the code base of the latest release does not match the text in the code that would generate this file if it wasn't there.  This causes the [Security Review module](https://github.com/backdrop-contrib/security_review/) to issue a failed test as documented [here](https://github.com/backdrop-contrib/security_review/issues/1)

## Steps To Reproduce

To reproduce the behaviour:
1. Download backdrop and install
2. Either check the code directly in files/.htaccess and core/includes/file.inc 539-555 or install security review and after setting the permissions to 444/rrr run it and check the "PHP files in the Backdrop files directory can be executed." test

## Actual behaviour
files/.htaccess
```
# Turn off all options we don't need.
Options None
Options +SymLinksIfOwnerMatch

# Set the catch-all handler to prevent scripts from being executed.
SetHandler Drupal_Security_Do_Not_Remove_See_SA_2006_006
<Files *>
  # Override the handler again if we're run later in the evaluation list.
  SetHandler Drupal_Security_Do_Not_Remove_See_SA_2013_003
</Files>

# If we know how to do it safely, disable the PHP engine entirely.
<IfModule mod_php5.c>
  php_flag engine off
</IfModule>
```

core/includes/file.inc 539-555
```
# Turn off all options we don't need.
Options -Indexes -ExecCGI -Includes

# Set the catch-all handler to prevent scripts from being executed.
SetHandler Drupal_Security_Do_Not_Remove_See_SA_2006_006
<Files *>
  # Override the handler again if we're run later in the evaluation list.
  SetHandler Drupal_Security_Do_Not_Remove_See_SA_2013_003
</Files>

# If we know how to do it safely, disable the PHP engine entirely.
<IfModule mod_php5.c>
  php_flag engine off
</IfModule>
<IfModule mod_php7.c>
  php_flag engine off
</IfModule>
```

## Expected behavior
Default .htaccess should match what is in file.inc and if permissions are set correctly the test in security review should pass.

## Additional information

Add any other information that could help, such as:
- Backdrop CMS version: 1.21.0
- Web server and its version: n/a
- Operating System and its version: n/a
- Browser(s) and their versions: n/a

### PR DESCRIPTION
**Pull request by yorkshire-pudding**
Fixes [Issue #5466](https://github.com/backdrop/backdrop-issues/issues/5466) Files directory htaccess does not match text in core/includes/file.inc 539-555

### CODE REVIEW

**Comment by yorkshire-pudding** [comment]
Checked in tugboat and security review test now passes.

**Comment by yorkshire-pudding** [comment]
Fixes https://github.com/backdrop-contrib/security_review/issues/1 "PHP files in the Backdrop files directory can be executed." test incorrect.

Not sure if this works in different orgs so may need manually closing once released

File: files/.htaccess [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Turn off all options we don't need.
-Options None
-Options +SymLinksIfOwnerMatch
+Options -Indexes -ExecCGI -Includes
```
Comment (oadaeh) [inline files/.htaccess:3]: If you're following D8/9, why do you not have -MultiViews in there?

Comment (yorkshire-pudding) [reply files/.htaccess:3]: @oadaeh - I'm not following D8/9 - I'm following what's in /core/includes/file.inc lines 539-555
https://github.com/backdrop/backdrop/blob/c144627cadb416d979aaa89bc9659c0cb04a7b2b/core/includes/file.inc#L539-L555
They need to match if security review test can be meaningful.

File: core/includes/file.inc [hunk 2]
```diff
@@ -536,6 +536,9 @@ function file_save_htaccess($directory, $private = TRUE, $force_overwrite = FALS
  */
 function file_htaccess_lines($private = TRUE) {
   $lines = <<<EOF
+# This code block is replicated in the file_htaccess_lines() function in
+# /core/includes/file.inc and files/.htaccess. If you update this file then 
+# make sure to also update the other location accordingly.
 # Turn off all options we don't need.
```
Comment (klonos) [inline core/includes/file.inc]: Can we please add a blank line above this one, in order to separate the comment that explains the code from the one re the replication of the files?

```suggestion

# Turn off all options we don't need.
```

File: core/includes/file.inc [hunk 3]
```diff
@@ -536,6 +536,9 @@ function file_save_htaccess($directory, $private = TRUE, $force_overwrite = FALS
  */
 function file_htaccess_lines($private = TRUE) {
   $lines = <<<EOF
+# This code block is replicated in the file_htaccess_lines() function in
+# /core/includes/file.inc and files/.htaccess. If you update this file then
```
Comment (klonos) [inline core/includes/file.inc]: There is a blank space at the end of this line, and also if/then in sentences should be separated with a comma:

```suggestion
# /core/includes/file.inc and files/.htaccess. If you update this file, then
```

File: files/.htaccess [hunk 4]
```diff
@@ -1,6 +1,8 @@
+# This code block is replicated in the file_htaccess_lines() function in
+# /core/includes/file.inc and files/.htaccess. If you update this file then
```
Comment (klonos) [inline files/.htaccess]: Same as with the other file:

```suggestion
# /core/includes/file.inc and files/.htaccess. If you update this file, then
```

File: files/.htaccess [hunk 5]
```diff
@@ -1,6 +1,8 @@
+# This code block is replicated in the file_htaccess_lines() function in
+# /core/includes/file.inc and files/.htaccess. If you update this file then 
+# make sure to also update the other location accordingly.
```
Comment (klonos) [inline files/.htaccess]: Add a blank line, in order to separate the two different comments:

```suggestion
# make sure to also update the other location accordingly.

```

**Review by klonos — APPROVED** [review]
LGTM 👍🏼  

Thank you so much @yorkshire-pudding for putting up with my OCD-driven scrutiny 🙏🏼

**Comment by yorkshire-pudding** [comment]
Closing in order to allow tests to be re-run as per Greg's advice

**Comment by yorkshire-pudding** [comment]
Re-open to allow tests to re-run

**Comment by quicksketch** [comment]
Merged into 1.x and 1.21.x. Thanks!

**Pull request merged**